integration_tests/dbt_project.yml

```diff
@@ -19,6 +19,9 @@ clean-targets:         # directories to be removed by `dbt clean`
     - "target"
     - "dbt_modules"
 
+vars:
+  dbt_utils_dispatch_list: ['dbt_utils']
+
```

Comment:
is this needed? I added it as part of my siege on Circle CI

Reply:
I don't think so.

This is only needed if `dbt_utils_integration_tests ` needed to override a `dbt_utils` macro, or inject some new macros into the `dbt_utils` namespace. That said, the integration test project does define [two schema tests](https://github.com/fishtown-analytics/dbt-utils/blob/master/integration_tests/macros/tests.sql),  `test_assert_equal` and `test_not_empty_string`; do you see any reason why we'd need to override those?

Reply:
the genesis for this PR was because we couldn't properly run the `dbt-utils`'s integration tests without being able to override the schema tests. my self-interested answer is: if t-sql can't run the integration tests's data tests, then we'll need a way to override them At a glance, I can't see how they could be an issue in TSQL, but I'm about to get my hands dirty to find out for sure

Reply:
Sounds good. If needed, we can rewrite those tests to be in the `dbt_utils` namespace (i.e. dispatchable from the integration tests) but not _actually_ ship with `dbt_utils` the package. That's essentially what I did for [`prep_external`](https://github.com/fishtown-analytics/dbt-external-tables/blob/master/integration_tests/macros/common/prep_external.sql) over in the `dbt-external-tables` integration tests.